frontend/src/app/shared/components/inner-tab-group/inner-tab-group.component.scss

```diff
@@ -8,7 +8,7 @@
     margin: 24px auto;
     border-radius: 8px;
 
-    background-color: lightgrey;
+    background-color: $color-lightgrey;
```

Comment:
nu as face peste `lightgrey` care e deja o variabila in sine o alta variabila `$color-lightgrey;`

Cred ca pentru cazul de **lightgrey**, ia un gri pe care il avem apropiat de asta si pune-l pe ala cat e -050 sau -060 sau cat e